_src/static/sass/utilities/_variables.scss

```diff
@@ -25,6 +25,8 @@ $baseline-horizontal:    $baseline;
 // color: config
 $transparent:       rgba(0, 0, 0, 0);
 
+$white:             rgb(255, 255, 255);
```

Comment:
Hmm, looks like this is already defined in https://github.com/edx/ux-pattern-library/blob/clrux/base-theme-settings/_src/static/sass/utilities/_variables.scss#L98. If the Sass Map makes it hard to customize this value, I'd recommend using the ``palette``function to define a ``$white`` var. Perhaps something like:

```scss
$white: palette(grayscale, white-t);
```